ch_tools/chadmin/cli/object_storage_group.py

```diff
@@ -101,10 +105,10 @@ def object_storage_group(ctx: Context, disk_name: str) -> None:
     help=("Use saved object list without traversing object storage again."),
 )
 @option(
-    "--store-state",
-    "store_state",
-    is_flag=True,
-    help=("Write total size of orphaned objects to log file."),
+    "--store-state-path",
+    "store_state_path",
+    default="",
+    help=("Zookeeper node path for storage total size of orphaned objects."),
```

Comment:
I think we need both options - store state in local file and store state in ZooKeeper. There are ClickHouse clusters with and without ZooKeeper.

Reply:
thanks, done